dashboards-reports/public/components/main/reports_table.tsx

```diff
@@ -79,7 +79,7 @@ const emptyMessageReports = (
             { defaultMessage: 'To learn more, see' }
           )}{' '}
           <EuiLink
-            href="https://docs-beta.opensearch.org/docs/opensearch-dashboards/reporting/"
+            href="https://opensearch.org/docs/docs/opensearch-dashboards/reporting/"
```

Comment:
![image](https://user-images.githubusercontent.com/32652829/125697077-b53230b4-6304-4d7f-a476-67059c0125fe.png)

Reply:
can you double check others